addons/l10n_ec_website_sale/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    'name': 'Ecuadorian Website',
+    'version': '1.0',
+    'category': 'Accounting/Localizations/Website',
+    'description': """Make ecommerce work for Ecuador.""",
+    'depends': [
+        'website_sale',
+        'payment',
```

Comment:
does not website_sale depend already? (via website_payment e.g.)

Reply:
Yes. website_sale > website_payment > account_payment > payment